client/driver/lxc.go

```diff
@@ -238,7 +238,7 @@ func (d *LxcDriver) validateVolumesConfig(volumes []interface{}) error {
 	for _, volDesc := range volumes {
 		volStr := volDesc.(string)
 		paths := strings.Split(volStr, ":")
-		if len(paths) != 2 {
+		if len(paths) < 2 || len(paths) > 3 {
```

Comment:
Do you want SplitN here if they're really paths, since : is a valid character in paths?